packages/twenty-server/src/engine/metadata-modules/remote-server/remote-table/remote-table.service.ts

```diff
@@ -567,4 +599,8 @@ export class RemoteTableService {
         throw new BadRequestException('Foreign data wrapper not supported');
     }
   }
+
+  private convertDistantTableColumnName = (columnName: string) => {
```

Comment:
I think I misunderstood your question early. We should not touch the column name. Only the table name is changed

Reply:
as discussed, kept and moved to an util